readme.md

```diff
@@ -30,48 +32,40 @@ Your app must:
 
 ### Necessary Deliverables
 
-* A **hosted copy of your game or app, built by you**, hosted on Github pages.
-  * Your HTML/CSS should pass the [HTML5 Validator](https://html5.validator.nu) and [CSS Validators](https://jigsaw.w3.org/css-validator/)
+* A **hosted copy of your game or app, built by you**, hosted on Github Pages
+  * Your HTML/CSS should pass the [HTML5 Validator](https://html5.validator.nu) and [CSS Validator](https://jigsaw.w3.org/css-validator/)
 * A **git repository hosted on Github** with frequent commits dating back to the very beginning of the project
   * A **link to your hosted project** in the URL section of your Github repo
-  * **A ``readme.md`` file** with explanations of the technologies used, the approach taken, installation instructions, unsolved problems, etc.
+  * **A ``readme.md`` file**, including
+    * Explanations of the technologies used, the approach taken, installation instructions, unsolved problems, etc.
+    * At least 5 [user stories]((https://github.com/ga-dc/curriculum/tree/master/02-intermediate-front-end/process-stories-wireframes).
 
 ---
 
 ### Deploying via GitHub Pages.
 
-**DO NOT** follow the instructions provided by GitHub for creating your Project Pages.  
-
-tldr;
+**DO NOT** follow the instructions provided by GitHub for creating your Project Pages. Why not? The instructions expect us to host our application on a custom web domain (yourname.com).
 
-1. Create a branch from "master" named "gh-pages".
-2. Push "gh-pages" to "origin".
-3. Browse to `<your_github_name>.github.io/<repo_name>`.
+Instead, try these steps:
 
-Why? They expect GitHub Project pages to provide an, easy way to create a separate information site for our application.  They expect us to host our application somewhere else (with robust servers, db, custom domain, etc.).  We just want our project to be available here.
-
-Steps:
-
-#### 1. Create a tracking branch, from master:
+#### 1. Create a gh-pages branch, from master:
 ```
 git checkout master
 git checkout -b gh-pages
 ```
 #### 2. Initial deploy to origin (via push):
 ```
-git push --set-upstream  # defaults to "origin gh-pages"
+git push -u origin gh-pages
 ```
 #### 3. Verify
-Browse to `<your_github_name>.github.io/<repo_name>`.  Don't forget to add this to your repo url and readme.
+Browse to `<your_github_username>.github.io/<repo_name>`
```

Comment:
Students regularly forgot to add it as their repo url and to their readme.  Why remove this comment?